networkx/algorithms/bipartite/tests/test_edgecoloring.py

```diff
@@ -0,0 +1,303 @@
+"""Unit tests for the :mod:`networkx.algorithms.bipartite.edge_coloring` module."""
+from collections import defaultdict
+
+import pytest
+
+import networkx as nx
+from networkx.algorithms.bipartite.edge_colorings import edge_coloring
+
+
+def _is_proper_edge_coloring(G, coloring):
+    """Checks through each node and saves the colors at each node to find out
+    if there is any conflict
+    """
+    node_colors = defaultdict(set)
+
+    # iterate through each edge in the graph
+
+    if isinstance(G, nx.MultiGraph):
```

Comment:
```suggestion
    if G.is_multigraph():
```
I think  this idiom appears elsewhere too.